app/models/mappers/check-activities.js

```diff
@@ -177,6 +188,10 @@ const getActivityLabelFromEvent = (event) => {
     return 'NOT YET DEFINED'
   }
 
+  if (event.activity?.activityType === 'received' && event.activity?.activityLabel.toLowerCase().includes('received')) {
```

Comment:
Is this to handle to 'old' audit records where there wqas an etxra 'received'? I fixed that so don;t think it will be an issue becuase all records produced in TEST onwards will have correct label of say 'Police correspondence form xyz police received'